com.obsproject.Studio.Plugin.DroidCam.metainfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.DroidCam</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>DroidCam OBS Plugin</name>
```

Comment:
I'd simplify this to `DroidCam` or `DroidCam Integration`. This name is shown in the "Extensions" section of the OBS Studio page of app stores, like GNOME Software, so we don't have to repeat the "OBS Plugin" information there.